bundles/org.openhab.binding.elroconnects/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,268 @@
+		</supported-bridge-type-refs>
+		<label>Door/Window Sensor</label>
+		<description>ELRO Connects door/window contact sensor</description>
+		<channels>
+			<channel id="entryalarm" typeId="entryalarm"/>
+			<channel id="entry" typeId="entry"/>
+			<channel id="battery" typeId="system.battery-level"/>
+			<channel id="lowbattery" typeId="system.low-battery"/>
+		</channels>
+		<representation-property>deviceId</representation-property>
+		<config-description>
+			<parameter name="deviceId" type="integer" required="true">
+				<label>Device ID</label>
+				<description>ID of the ELRO Connects Device.</description>
+			</parameter>
+		</config-description>
+	</thing-type>
+	<thing-type id="motionsensor">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="connector"/>
+		</supported-bridge-type-refs>
+		<label>Motion sensor</label>
+		<description>ELRO Connects motion sensor</description>
+		<channels>
+			<channel id="motionalarm" typeId="motionalarm"/>
+			<channel id="motion" typeId="system.motion"/>
+			<channel id="battery" typeId="system.battery-level"/>
+			<channel id="lowbattery" typeId="system.low-battery"/>
+		</channels>
+		<representation-property>deviceId</representation-property>
+		<config-description>
+			<parameter name="deviceId" type="integer" required="true">
+				<label>Device ID</label>
+				<description>ID of the ELRO Connects Device.</description>
+			</parameter>
+		</config-description>
+	</thing-type>
+	<thing-type id="temperaturesensor">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="connector"/>
+		</supported-bridge-type-refs>
+		<label>Motion sensor</label>
+		<description>ELRO Connects motion sensor</description>
+		<channels>
+			<channel id="temperature" typeId="temperature"/>
+			<channel id="humidity" typeId="system.atmospheric-humidity"/>
+			<channel id="battery" typeId="system.battery-level"/>
+			<channel id="lowbattery" typeId="system.low-battery"/>
+		</channels>
+		<representation-property>deviceId</representation-property>
+		<config-description>
+			<parameter name="deviceId" type="integer" required="true">
+				<label>Device ID</label>
+				<description>ID of the ELRO Connects Device.</description>
+			</parameter>
+		</config-description>
+	</thing-type>
+	<thing-type id="powersocket">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="connector"/>
+		</supported-bridge-type-refs>
+		<label>Motion sensor</label>
+		<description>ELRO Connects power socket</description>
+		<channels>
+			<channel id="powerstate" typeId="system.power"/>
+		</channels>
+		<representation-property>deviceId</representation-property>
+		<config-description>
+			<parameter name="deviceId" type="integer" required="true">
+				<label>Device ID</label>
+				<description>ID of the ELRO Connects Device.</description>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<channel-type id="scene">
+		<item-type>String</item-type>
+		<label>Scene</label>
+		<description>Scene</description>
```

Comment:
Could you add a little more to the description to give a more in depth detail beyond what the label is?